main.js

```diff
@@ -439,7 +453,9 @@
         };
 
         items.forEach(function(item) {
-            itemQueue.push(item);
+            setTimeout(function(){
+                itemQueue.push(item);
+            }, 500); // have some healthy delay or steam will block you for flooding
         });
```

Comment:
Think about what exactly that items.foreach loop does
Before:
Add all items to list

After:
Start a setTimeout per item
wait 500ms
Add all items to the queue